### DISCUSSION TITLE
Is this library still maintained?

### DISCUSSION THREAD

**Discussion by lucksp** [react-native-elements/react-native-elements#3944] [General]
I am curious if this library is still maintained?  I see the last merged PR was in January - 8 months ago...

**Comment by VetSoftKeith** [comment]
I've been starting to wonder that myself.  The latest release has been an RC for at least a year sine I started using it.

**Comment by muhammadashfaq** [comment]
@lucksp I am wondering the library which is the first preference for me as a react-native-developer is not being maintained since one year.

**Comment by ChromeQ** [comment]
Seems like this is dead, what do others recommend as an alternative?

**Comment by theianmay** [comment]
## 🚀 React Native Elements v5 Beta is Live!  

Hello everyone! 👋 Thank you all for your patience while our new team has been hard at work. 
We’re excited to announce that **v5.0.0-beta.1** is now available! 

This is a **beta release**, intended for testing and feedback. The **full stable v5.0.0** is still in the pipeline.  

---
###⚠️ What you need to know
- This is not yet stable, intended for testing and feedback.
- For example, a known issue is: Button disabled state styling on Android. Workarounds and discussion available here: #3985
- Associated issue: https://github.com/facebook/react-native/issues/53849
- You can install the beta via NPM:
```bash
npm install @rneui/base@beta @rneui/themed@beta
```
---

### 📝 Feedback
Please provide **bugs, suggestions, or feedback**:  
- GitHub Discussion: https://github.com/react-native-elements/react-native-elements/discussions/4007 
- Discord: https://discord.com/invite/e9RBHjkKHa

**Next Steps:**  
- I will be closing this for now to keep things organized.
- All ongoing beta feedback should go to the Discussion or Discord.
